project_argparser.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+
+#Creating the parser
+parser = argparse.ArgumentParser(description="Electric car racing simulation")
+
+#This class is meant to make it easier to enter two correct inputs and check against them
+#Preferably the name you give to an object of this class would be the same (or as close to the same without conflict) as the name of the argument created within it
+#...to allow for easily calling arg_check() (ex:logging_arg.arg_check(args.logging))
```

Comment:
As a coding standard I generally like putting imports and declarations (e.g. class defs) before the executable code in file. Perhaps it's an artifact of being an old C/C++ programmer but it allows a little cleaner consistent file organization and flows through the executable code.

Reply:
Understood -- I'll make sure to keep class defs, etc, above from now on.